src/config/config.ts

```diff
@@ -454,6 +451,36 @@ export class Configuration {
   };
 
   // --- GETTERS --- //
+  getAllAddressFormat(): string {
+    const bitcoin = this.getBitcoinAddressFormat();
+    const lightning = this.getLightningAddressFormat();
+    const ethereum = this.getEthereumAddressFormat();
+    const cardano = this.getCardanoAddressFormat();
+    const defichain = this.getDefichainAddressFormat();
+
+    return `${bitcoin}|${lightning}|${ethereum}|${cardano}|${defichain}`;
+  }
+
+  getBitcoinAddressFormat(): string {
+    return '([13]|bc1)[a-zA-HJ-NP-Z0-9]{25,62}';
+  }
+
+  getLightningAddressFormat(): string {
+    return '(LNURL|LNDHUB)[A-Z0-9]{25,250}|LNNID[A-Z0-9]{66}';
+  }
+
+  getEthereumAddressFormat(): string {
+    return '0x\\w{40}';
+  }
+
+  getCardanoAddressFormat(): string {
+    return 'stake[a-z0-9]{54}';
+  }
+
+  getDefichainAddressFormat(): string {
+    return this.environment === Environment.PRD ? '8\\w{33}|d\\w{33}|d\\w{41}' : '[78]\\w{33}|[td]\\w{33}|[td]\\w{41}';
+  }
+
```

Comment:
Why not use properties instead of methods?

```ts
  allAddressFormat = `${bitcoinAddressFormat}|${lightningAddressFormat}|${ethereumAddressFormat}|${cardanoAddressFormat}|${defichainAddressFormat}`;

  bitcoinAddressFormat = '([13]|bc1)[a-zA-HJ-NP-Z0-9]{25,62}';
  lightningAddressFormat = '(LNURL|LNDHUB)[A-Z0-9]{25,250}|LNNID[A-Z0-9]{66}';
  ...
```
And maybe nest it into an addressFormat object?

Reply:
"_Why not use properties instead of methods?_" - good question - maybe because I am still in the JAVA world ;-)